.github/workflows/pytest_coverage.yml

```
@@ -0,0 +1,60 @@
+name: pytest and coverage report
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)